testnets/sgenetworktestnet/assetlist.json

```diff
@@ -0,0 +1,29 @@
+{
+  "$schema": "../../assetlist.schema.json",
+  "chain_id": "sgenetworktestnet",
+  "assets": [
+    {
+      "description": "The SGE token is primarily a governance token for the SGE Network blockchain.",
+      "denom_units": [
+        {
+          "denom": "usge",
+          "exponent": 0,
+          "aliases": []
+        },
+        {
+          "denom": "usge",
+          "exponent": 6,
+          "aliases": []
+        }
+      ],
+      "base": "usge",
+      "name": "SGE Network",
+      "display": "sge",
+      "symbol": "SGE",
+      "logo_URIs": {
+        "png": "./images/logo.png",
+        "svg": "./images/logo.svg"
```

Comment:
for the token logos, please upload images with square dimensions. The logotypes you've uploaded are fine for Chain images, but not for Asset images.

Reply:
For the chain logos, those logos appropriate. But for the asset logos, yes you technically made the files square, but we don't want the entire logotype with the wording 'Sports Gaming and Entertainment Network', but only the logo asset--the golden 'S' thing--which would be appropriate as an actual token design.

Reply:
@JeremyParish69 Please review now.

Reply:
Thanks for this. The SVG token logo is a rather poor quality conversion from the png. Could we either remove it and only use the png, or else grab the icon from the 'logo.svg' image (which doesn't have the subtle reflections in it) and use that for both?

Reply:
@JeremyParish69 Updated the PR